lib/std/unicode.zig

```diff
@@ -153,6 +154,50 @@ pub fn utf8Decode4(bytes: []const u8) Utf8Decode4Error!u21 {
     return value;
 }
 
+/// Returns true if the given unicode codepoint can be encoded in UTF-8.
+pub fn utf8ValidCodepoint(value: u21) bool {
```

Comment:
This is not specific to utf-8.

Reply:
Copy pasted from PR. :)
What is better name? `isValidCodepoint`?

Reply:
Sounds good. Make sure to also edit the comment.